.github/workflows/python-package.yml

```diff
@@ -7,13 +7,32 @@ on:
     branches: [master, main]
 
 jobs:
-  build:
+  build-lint-unit-test:
     runs-on: ubuntu-latest
     strategy:
       matrix:
         python-version: ['3.9', '3.10', '3.11', '3.12', '3.13']
-      # Limit parallel runs on pushes to master to 1 to limit integration test rate limited failures
-      max-parallel: ${{ github.event_name == 'pull_request' && 5 || 1 }}
+      max-parallel: 1
```

Comment:
Did you mean to make the linting job as max-parallel 1 as well?

Reply:
I didn't, but its fixed now.